templates/default/client.rb.erb

```diff
@@ -40,7 +40,7 @@ https_proxy "<%= node["chef_client"]["config"]["https_proxy"] %>"
 no_proxy "<%= node["chef_client"]["config"]["no_proxy"] %>"
 <% end -%>
 
-<% if node.attribute?("ohai") && node["ohai"].attribute?("plugin_path") -%>
+<% unless node["ohai"]["plugin_path"].nil? -%>
```

Comment:
You could write `if node["ohai"]["plugin_path"]` as `nil` and `false` are the only *false* values in Ruby. I do see the above more often in Chef recipes and my suggestion more in Ruby code.